Initial Rolf documentation

axelbsa opened the pull request.

oskov-usn-no committed:
Added templates for Digdir FReg and KRR API, digdir.md

glorytiger committed:
Update jwt-mot-backend.md

La til bemerkelse om at dette er en generell oppskrift med en link til den spesifike for Maskinporten. For de som klikker seg inn hit før de har sett sistnevnte.
Endret beskrivelse om autentisering til og også omhandle asymmetriske nøkler da det er disse vi benytter i IntArk nå.

axelbsa committed:
Initial Rolf documentation

jokim commented on docs/datadeling/teknisk-plattform/api/rolf-api.md:
```diff
@@ -0,0 +1,39 @@
+title: Oppsett i Gravitee for Rolf SA
+---
+
+Rolf SA Api er en påkrevd delkomponent i UH-SAK.
```
Foreslår ei beskriving som forklarer litt meir utdjupande, sidan det kan bli lest av folk som ikkje er interne i UH-sak-prosjektet. Til dømes:
> Rolf er et rolleregister, som blant annet brukes av UH-sak (sak og arkiv) for å forvalte personers forretningsroller, som brukes for UH-sak sin tilgangsstyring. "Rolf SA API" er Rolf sitt API som er det som brukes av UH-sak.
> Rolf driftes av UiO, se nettsidene [Rolf - sektorverktøy for systematisk bruk av roller](https://www.uio.no/tjenester/it/adm-app/rolf/) for mer informasjon.

jokim commented on docs/datadeling/teknisk-plattform/api/rolf-api.md:
```diff
@@ -0,0 +1,39 @@
+
+Kortversjon:
+
+1. Log inn i Gravitee som administrator.
```
But first: 1. Bestill oppsett av Rolf fra UiO, se [Bestill Rolf](https://www.uio.no/tjenester/it/adm-app/rolf/bestill-rolf.html). Du vil få tilsendt en JSON-fil du kan bruke videre for oppsettet.

jokim commented on docs/datadeling/teknisk-plattform/api/rolf-api.md:
```diff
@@ -0,0 +1,39 @@
+2. Opprett et nytt API ved å trykke på pluss-knappen nede til høyre.
+3. Trykk på "IMPORT"-knappen
+4. Velg tilsendt .json fil
+5. Trykk på "API out of sync, deploy your API" langs toppen.
```
Er det ingen manuelle justeringar du vil ønske som institusjon? Til dømes legge på fleire policies eller plans?

jokim commented on docs/datadeling/teknisk-plattform/api/rolf-api.md:
```diff
@@ -0,0 +1,39 @@
+7. Når du er klar, trykk, Publish The API, under General - Details
+
+
+## Hvordan importere API fra vedsendt json fil
```
Kva er "vedsendt"? Tilsendt?

axelbsa replied:
Det er ikke engang et ord på dansk.

jokim commented on docs/datadeling/teknisk-plattform/api/rolf-api.md:
```diff
@@ -0,0 +1,39 @@
+## Hvordan importere API fra vedsendt json fil
+
+- Opprett et nytt API ved å trykke på pluss-knappen nede til høyre.
+![Importere API fra json](/datadeling/img/rolf-new-api_arrow.png)
```
Bildet ser ut til å fungere for dei som har 4K super-widescreen. Foreslår å zoome litt inn før du tek skjermskot, så det vert enklare å sjå for oss som sit på liten laptop :)

adrfell committed:
Update maskinporten.md with / missing from end of DFO test url

There was a / missing at the end of the DFO test url.

axelbsa committed:
Fixes

jsama-uio committed:
Adjust wording in DFØ notification documentation

Attempted to make it clearer how to set up transfer of notifications
from DFØ to the institution.

ponas committed:
datadeling: update URLs to the FS-API test and development environments

vidarstokke committed:
Update info-ved-eskalering.md

axelbsa committed:
Update rolf-api.md

Remove newline in title front matter

Frosco committed:
Prepared for docusaurus update

Frosco committed:
Updated packages

Frosco committed:
Fixed a broken link

Frosco committed:
Updated prop types according to how they are used

Frosco committed:
Deleted unnecessary files
